tests/sentry/tasks/test_update_user_reports.py

```diff
@@ -37,29 +38,35 @@ def test_simple(self):
         assert report3.group_id is None
         assert report3.environment_id is None
 
-    def test_start_end(self):
+    def test_start_end_reports(self):
```

Comment:
I see that `test_start_end_reports()` and `test_start_end_events()` have a lot of common and repetitive code. Can that common code be moved to a function?